Core/Core/SubmitAssignmentExtension/ViewModel/CoursePickerViewModel.swift

```diff
@@ -16,13 +16,16 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 //
 
+import Combine
 import SwiftUI
 
 public class CoursePickerViewModel: ObservableObject {
     public typealias Course = IdentifiableName
 
     @Published public var selectedCourse: CoursePickerViewModel.Course?
     @Published public var state: ViewModelState<[Course]> = .loading
+    public private(set) lazy var dismissView: AnyPublisher<Void, Never> = dismissViewSubject.eraseToAnyPublisher()
+    private let dismissViewSubject = PassthroughSubject<Void, Never>()
```

Comment:
For simplicity's sake, let's expose the `PassthroughSubject` to have less boilerplate.

Reply:
Fixed!